components/brave_ads/core/internal/ml/data/vector_data.cc

```diff
@@ -141,6 +142,35 @@ float operator*(const VectorData& lhs, const VectorData& rhs) {
   return dot_product;
 }
 
+float VectorData::Multiply(const ::flatbuffers::Vector<float>& lhs) {
```

Comment:
Why are we adding new VectorData functions?

Reply:
reused original functions